content/applications/inventory_and_mrp/purchase/manage_deals/control_bills.rst

```diff
@@ -2,127 +2,143 @@
+banner, the :guilabel:`Should Be Paid` field status is set to :guilabel:`No`.
 
 .. tip::
-   The :guilabel:`Should Be Paid` status on bills is set automatically by Odoo. However, the status
-   can be changed manually by clicking the field's drop-down menu inside the :guilabel:`Other Info`
+   The :guilabel:`Should Be Paid` status on bills is automatically set by Odoo. However, the status
+   can be manually changed by clicking the field's drop-down menu inside the :guilabel:`Other Info`
    tab.
 
 View a purchase order's billing status
 ======================================
 
-When a purchase order is confirmed, its :guilabel:`Billing Status` can be viewed under the
-:guilabel:`Other Information` tab on the purchase order form.
+Once a |PO| is confirmed, its :guilabel:`Billing Status` can be viewed under the :guilabel:`Other
+Information` tab on the |PO| form.
+
+To view the :guilabel:`Billing Status` of a |PO|, navigate to :menuselection:`Purchase app -->
+Orders --> Purchase Orders`, and select a |PO| to view.
+
+Click the :guilabel:`Other Information` tab, and locate the :guilabel:`Billing Status` field.
 
-.. image:: control_bills/billing-status-nothing-to-bill.png
+.. image:: control_bills/control-bills-billing-status.png
    :align: center
-   :alt: Purchase order billing status.
-
-Below is a list of the different statuses that a :guilabel:`Billing Status` could appear as and when
-they are displayed, depending on the bill control policy used.
-
-+------------------------------+--------------------------------------------------------------------------+
-| :guilabel:`Billing Status`   |                              **Conditions**                              |
-|                              +------------------------------------------------+-------------------------+
-|                              |            *On received quantities*            | *On ordered quantities* |
-+------------------------------+------------------------------------------------+-------------------------+
-| :guilabel:`Nothing to Bill`  |       PO confirmed; no products received       |     *Not applicable*    |
-+------------------------------+------------------------------------------------+-------------------------+
-| :guilabel:`Waiting Bills`    |  All/some products received; bill not created  |       PO confirmed      |
-+------------------------------+------------------------------------------------+-------------------------+
-| :guilabel:`Fully Billed`     | All/some products received; draft bill created |    Draft bill created   |
-+------------------------------+------------------------------------------------+-------------------------+
+   :alt: Billing status field on a purchase order form.
+
+The table below details the different values the :guilabel:`Billing Status` field could read, and
+when they are displayed, depending on the *bill control* policy used.
```

Comment:
```suggestion
when they are displayed, depending on the *Bill Control* policy used.
```
I understand that asterisks/italics are being used here, because it's not appearing directly on the UI at this point. However, since it is a properly-named feature, do you think capitalizing it is the right move. This is just my suggestion. Curious to know your thoughts

Reply:
Yes, that makes sense to me. Suggestion implemented ✅